tools/mmp/driver.cs

```diff
@@ -442,6 +442,29 @@ static void Main2 (string [] args)
 
 			}
 
+			if (IsUnified) {
+				List<ExclusionItem> exclusionItems = ReadExclusionList ();
+				List<string> referencesToFix = new List<string> ();
+
+				foreach (var asm in references) {
+					List<ExclusionItem> possibleMatches = exclusionItems.Where (x => x.Name == Path.GetFileName (asm)).ToList ();
+					if (possibleMatches.Count > 0) {
+						var module = ModuleDefinition.ReadModule (asm);
```

Comment:
that's done outside the resolver :| so it kills all caching :(
even more of a reason to exclude modern/mobile